lib/Conversion/TTIRToTTNN/TTIRToTTNN.cpp

```diff
@@ -199,7 +260,9 @@ void populateTTIRToTTNNPatterns(MLIRContext *ctx, RewritePatternSet &patterns,
            EmbeddingOpConversionPattern,
            SoftmaxOpConversionPattern,
            TransposeOpConversionPattern,
-           MatmulOpConversionPattern
+           SoftmaxOpConversionPattern,
+           MatmulOpConversionPattern,
+           Conv2dOpConversionPattern
```

Comment:
Is there no maxpool conversion?

Reply:
Not yet, I'll bringup the maxpool op when tt-metal gets uplifted. Should I delete the op definitions from this branch?